robot/basestation/static/js/elements/feed/cmd_input.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(() => {
+  const send_cmd = $('#send-serial-cmd')
+  const cmd_input = $('#serial-cmd-input')
+
+  function handleCMD() {
+    const full_cmd = cmd_input.val()
+    const prefix = full_cmd.split(' ').shift()
+    const cmd = full_cmd.split(' ').pop()
+    if (cmd != '' && cmd != full_cmd) {
+      sendCMD(prefix, cmd)
+    }
+
+    else {
+      appendToConsole('no command to send')
```

Comment:
Instead of saying this, it should be an explanation on how to use the prefix suffix system

Reply:
Fixed